README.md

```diff
@@ -19,7 +19,7 @@ export ZONE=us-east1-b
 export AUTH_FILE=<path to service-account.json>
 
 ./prombench gke cluster create -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID \
-    -v ZONE:$ZONE -v CLUSTER_NAME:$CLUSTER_NAME -f manifests/cluster.yaml
+    -v ZONE:$ZONE -v CLUSTER_NAME:$CLUSTER_NAME -f manifests/cluster-infra/0_cluster.yaml
```

Comment:
now the Readme can also be simplified.

Reply:
oh i just noticed the commands are different for deploying the `cluster.yaml` and the other k8s objects. as `cluster.yaml` is for gke. should we revert the change?

Reply:
can't we deploy these in a single step?

Reply:
aaah I see now, these are different commands - cluster clreate vs resource apply.

Reply:
I updated it now.